### DISCUSSION TITLE
which sdk interface obtains the buckets capacity?

### DISCUSSION THREAD

**Discussion by metadeng** [minio/minio#16747] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
when I use the S3 SDK for Java to count the size of each bucket, I find that none of the interfaces fit? also tried the minio SDK, still no!

**Describe the solution you'd like**
The minio SDK provides an interface to get the current capacity of each bucket

**Comment by klauspost** [comment, marked as answer]
You say you want "size of each bucket" and "current capacity of each bucket"

They are different things, but either way the S3 API provides neither.

The [minio admin](https://github.com/minio/madmin-go) API provides these, but we only provide that in Go and with no compatibility guarantees.

[mc](https://min.io/docs/minio/linux/reference/minio-mc.html) cli provides usage functionality as well through `mc stat`.
